docs/ContribOperators.md

```diff
@@ -131,6 +131,8 @@ This version of the operator has been available since version 1 of the 'com.micr
 <dd>Corresponding past and present are same tensor, its size is (2, batch_size, num_heads, max_sequence_length, head_size)</dd>
 <dt><tt>qkv_hidden_sizes</tt> : list of ints</dt>
 <dd>Hidden dimension of Q, K, V: hidden_size, hidden_size and v_hidden_size</dd>
+<dt><tt>softmax_scale</tt> : float</dt>
+<dd>Custom softmax_scale will be used otherwise softmax_scale = 1/sqrt(head_size). Default value is 0.0f</dd>
```

Comment:
Regarding to "default value 0.0f".  I think the attribute is optional. If not specified, the default value is 1/sqrt(head_size). Right?

Reply:
fixed